products/cakephp.md

```diff
@@ -0,0 +1,81 @@
+---
+title: CakePHP
+permalink: /cakephp
+alternate_urls:
+  - /cake-php
+  - /cake
+category: framework
+iconSlug: cakephp
+releasePolicyLink: https://book.cakephp.org/4/en/release-policy.html
+releaseLabel: "CakePHP __RELEASE_CYCLE__ __CODENAME__"
+releaseColumn: true
+releaseDateColumn: true
+activeSupportColumn: true
+discontinuedColumn: false
+versionCommand: php bin/cake.php version
+
+auto:
+  - git: https://github.com/cakephp/cakephp.git
+
+releases:
+  - releaseCycle: "4.4"
+    eol: false
+    support: true
+    releaseDate: 2019-12-15
+    latest: "4.4.5"
+    latestReleaseDate: 2022-08-28
+    link: https://bakery.cakephp.org/2019/12/15/cakephp_400_released.html
+    codename: Strawberry
+  - releaseCycle: "3.10"
+    eol: 2022-12-15
+    support: 2021-12-15
+    releaseDate: 2021-06-20
+    latest: "3.10.4"
+    latestReleaseDate: 2022-05-21
+    link: https://bakery.cakephp.org/2021/06/19/cakephp_3100_released.html
+  - releaseCycle: "3.9"
+    eol: true
+    support: false
+  - releaseCycle: "3.8"
+    eol: true
+    support: false
+  - releaseCycle: "3.7"
+    eol: true
+    support: false
+  - releaseCycle: "3.6"
+    eol: true
+    support: false
+  - releaseCycle: "3.5"
+    eol: true
+    support: false
+  - releaseCycle: "3.4"
+    eol: true
+    support: false
+  - releaseCycle: "3.3"
+    eol: true
+    support: false
+  - releaseCycle: "3.2"
+    eol: true
+    support: false
+  - releaseCycle: "3.1"
+    eol: true
+    support: false
+  - releaseCycle: "3.0"
```

Comment:
each releasecycle should have at least releaseDate tag

Reply:
OK, I'm going to add them

Reply:
thank you 👍 💯 🥇 🎉 
and if you have time and of course if you could find it , could you also change eol tags to date instead of using true
[sure if you couldnt find them you can leave it as true but we prefer specific dates]

Reply:
After this change it is ok to merge so all green after this one :)

Reply:
> After this change it is ok to merge so all green after this one :)

OK, but I am gonig to add some more stuff, if you don't mind.

Reply:
Sure :) i was talking about min to be landed more means better :)

Reply:
btw if cake doesnt have eol dates you can use support dates as its eol dates instead of using true

Reply:
OK, thanks. But it does. They aren't very clearly documented in all cases, though.